components/Portfolio/Trading/Overview/Equity/index.tsx

```diff
@@ -33,15 +31,14 @@ const ToggleTable = () => {
 
 interface EqProps {
     className?: string;
-    balances: UserBalance;
-    filledOrders: FilledOrder[];
-    fairPrice: BigNumber;
-    baseTicker: string;
-    quoteTicker: string;
+    balances?: UserBalance;
+    filledOrders?: FilledOrder[];
+    fairPrice?: BigNumber;
+    baseTicker?: string;
+    quoteTicker?: string;
```

Comment:
Typescript loses its value if you set all of them to optional. Id suggest reverting these changes and making sure the value passed in exists. ie balances={balances ?? defaults.balances}